packages/ketcher-react/src/script/ui/dialog/mainmenu/save.jsx

```diff
@@ -43,9 +48,9 @@ const saveSchema = {
     },
     format: {
       title: 'Format',
-      enum: Object.keys(structFormat.map),
-      enumNames: Object.keys(structFormat.map).map(
-        fmt => structFormat.map[fmt].name
+      enum: Object.keys(SupportedFormatPropertiesMap),
+      enumNames: Object.keys(SupportedFormatPropertiesMap).map(
+        fmt => SupportedFormatPropertiesMap[fmt].name
```

Comment:
Please, don't shortcut `format` word to `fmt`. It is not so long to do so.

Reply:
Same for `options` -> `opts`. It doesn't make sence. Just 3 symbol less

Reply:
fixed